ganttproject/data/resources/language/i18n.properties

```diff
@@ -616,6 +616,7 @@ task.move.up = Up
 task.move.up.description = Move task up
 task.new = New task
 task.notesTooltip.pattern = <html>{0}</html>
+task.changeProgressHint = {0}% ({1}d)
```

Comment:
should TaskDuration be in hours, it would insert `h` automatically. Just gettign rid of code, configs or whatever else which think that duration is measured in days.